api/src/wfl/service/slack.clj

```diff
@@ -0,0 +1,41 @@
+(ns wfl.service.slack
+  "Interact with Slack API."
+  (:require [clojure.data.json :as json]
+            [clojure.string :as str]
+            [clj-http.client :as http]
+            [wfl.environment :as env]
+            [wfl.util :as util]))
+
+(defn ^:private api-url
+  "API URL for Slack."
+  [& parts]
+  (let [url "https://slack.com/api"]
+    (str/join "/" (cons url parts))))
+
+;; Slack Bot User token obtained from Vault
+(defonce ^:private token
+  (delay (:bot-user-token (#'env/vault-secrets "secret/dsde/gotc/dev/wfl/slack"))))
```

Comment:
Is this path consistent across environments (I saw `…gotc/dev…` and thought of the prod env)?

Reply:
This is a really good point! I think we will stick to one Slack token for all environments (we also have one slack Bot and one slack Org), but if it brings confusion, maybe we should move it to a different Vault path? (such as `secret/dsde/gotc/wfl`? ) I realized we have similar issues with other token/credentials as well…